transportable-udfs-codegen/src/main/java/com/linkedin/transport/codegen/SparkWrapperGenerator.java

```diff
@@ -49,13 +56,15 @@ private void generateWrapper(String topLevelClass, Collection<String> implementa
     ClassName wrapperClass =
         ClassName.get(topLevelClassName.packageName() + "." + SPARK_PACKAGE_SUFFIX, topLevelClassName.simpleName());
     String udfImplementationInstantiations = implementationClasses.stream()
-        .map(clazz -> "new " + clazz + "()")
+        .map(clazz -> "new " + clazz + parameters(clazz, classToNumberOfTypeParameters) + "()")
```

Comment:
I am not sure I understand the need for this. Nothing has changed in the Spark StdUDFWrapper and this change requires addition of more metadata.

Reply:
The UDF classes themselves can be parameterized now.  See ArrayFillFunction.